lib/src/main/java/io/ably/lib/util/Crypto.java

```diff
@@ -111,12 +129,10 @@ public static CipherParams getDefaultParams(String base64Key) {
 	}
 
 	/**
-	 * Obtain default CipherParams using key and algorithm from other CipherParams object
-	 * @param params
-	 * @return
+	 * Package scoped method for unit testing purposes.
```

Comment:
This is a public API: https://docs.ably.io/client-lib-development-guide/features/#RSE1